detect:
  # should be true - (line_count == the number of y_offset elements)
  line_count: 1   # the number of detection lines that will be used
  y_offset:
    - 25
  continuous_pixel: 30   # FIXME: the number of pixels that will be considered as one lane pixel
  offset:
    left: 0
    right: 0